src/main/java/org/app/controller/TodoController.java

```diff
@@ -14,7 +14,8 @@
 @AllArgsConstructor
 @Validated
 @RequestMapping("/api/user")
-public class TodoController {
+public class TodoController { // todo
```

Comment:
Add an ability for user to view all his unfinished tasks.
Add filtering
- See tasks for specific date
- Include finished tasks